modules/core/src/main/java/org/apache/ignite/internal/suggestions/OsConfigurationSuggestions.java

```diff
@@ -61,11 +61,11 @@
         List<String> suggestions = new ArrayList<>();
 
         if (U.isRedHat()) {
```

Comment:
Can we perhaps do this for all Linux VMs and not just Red Hat? I have Ubuntu but all of these VM params are there.